internal/worker/worker.go

```diff
@@ -106,13 +106,16 @@ func (w *Worker) ProcessRawImage(ctx context.Context, reference oci.Reference) e
 
 	versionDiffSortable := semver.PackInt64(nil)
 
-	// If no new version was defined and the current version is using the "latest"
-	// convention, the latest available reference is the current reference
-	if reference.Version() == "latest" {
-		r := reference
-		data.LatestReference = &r
-		versionDiffSortable = 0
-	}
+	// TODO: This should be removed, right?
```

Comment:
TODO

Reply:
<img width="861" alt="image" src="https://github.com/user-attachments/assets/4d3f80c6-7215-439b-a8d4-02c807ae6f9e" />

If only the digest is different, set the diff to 0 rather than nil?

Reply:
Fixed.